packages/docs/components/GoogleAnalytics.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+'use client';
+
+import Script from 'next/script';
+
+const GA_MEASUREMENT_ID = process.env.NEXT_PUBLIC_GA_ID;
```

Comment:
Any real harm in hardcoding this inline?

Reply:
No, it's public in the client anyways. I'll update this and remove the vercel env var